packages/mobile/src/identity/reducer.ts

```diff
@@ -349,7 +362,42 @@ export const secureSendPhoneNumberMappingSelector = (state: RootState) =>
 export const importContactsProgressSelector = (state: RootState) =>
   state.identity.importContactsProgress
 export const matchedContactsSelector = (state: RootState) => state.identity.matchedContacts
-export const verificationStateSelector = (state: RootState) => state.identity.verificationState
+
+export const isBalanceSufficientForSigRetrievalSelector = (state: RootState) => {
+  const dollarBalance = stableTokenBalanceSelector(state) || 0
+  const celoBalance = celoTokenBalanceSelector(state) || 0
+  return isBalanceSufficientForSigRetrieval(dollarBalance, celoBalance)
+}
+
+function isBalanceSufficientForAttestations(state: RootState, attestationsRemaining: number) {
+  const userBalance = stableTokenBalanceSelector(state) || 0
+  return new BigNumber(userBalance).isGreaterThan(
+    attestationsRemaining * ESTIMATED_COST_PER_ATTESTATION
+  )
+}
```

Comment:
This was copied from the existing code, but it looks like this is gonna fail for users with only CELO balances, right?

Reply:
Yeap, afaik we require cUSD to pay for attestations